### PR TITLE
Test merge

### PR DESCRIPTION
**Pull request by arvidn**

### CODE REVIEW

**Comment by arvidn** [comment]
@aldenml @ssiloti I can't understand where this (consistent) call to a pure virtual function in the python binding comes from. I cannot reproduce it on my mac.

**Comment by aldenml** [comment]
looked at it and I can't help :( not familiar enough with the python binding

**Comment by ssiloti** [comment]
I am able to repro this on my linux machine. Here's a [backtrace](https://gist.github.com/ssiloti/bc357169136980a4c66bd977f99c8c82). This is with Boost 1.65.1.

**Comment by arvidn** [comment]
thanks. I didn't even try it on my linux machine (since it happened on osx on travis). I will try that too

**Comment by ssiloti** [comment]
4e7cb26 fixed the crash for me.

**Comment by arvidn** [comment]
the remaining problem appears to be that dependency according to boost-build is boost_python-vc140-mt-1_63.dll whereas the actual dependency (as reported by dumpbin) is boost_python3-vc140-mt-1_63.dll

(i.e. "python3" instead of "python"). I'll take a look at how RC_1_1 differs here

File: src/session_impl.cpp [hunk 1]
```diff
@@ -5795,20 +5815,23 @@ namespace aux {
...
-			if (m_outstanding_router_lookups == 0) update_dht();
+			if (m_outstanding_router_lookups == 0) start_dht();
 			return;
 		}
 
 
 		for (auto const& addr : addresses)
 		{
+#if !TORRENT_USE_IPV6
+			if (!i->is_v4()) continue;
```
Comment (aldenml) [inline src/session_impl.cpp]: where is this `i` defined?, could it be `addr` instead?

**Pull request merged**